### DISCUSSION TITLE
How can I fix the error 'No existing Nx Cloud client and failed to download new version' when building nx?

### DISCUSSION THREAD

**Discussion by wcwcaseman** [nrwl/nx#28499] [Questions]
Hi, I would like to run NX locally and point my repo to a local build. However when I run pnpm local-registry I get a build error that states 'No existing Nx Cloud client and failed to download new version'.  I'm using the master branch without any code changes. I have run pnpm install, i'm running node 18.18 and pnpm 8.10.5.

**Comment by sameer98441** [comment]
I tried `pnpm i -g nx-cloud`, which did the trick. Works now on mine.
